spec/features/admin_adds_a_group_spec.rb

```diff
@@ -9,7 +9,7 @@
 
     fill_in Group.human_attribute_name(:name), with: "St. Peter"
     fill_in Group.human_attribute_name(:responsable_name), with: "Mr. Black"
-    fill_in Group.human_attribute_name(:responsable_contact), with: "0045-123-4445"
+    fill_in Group.human_attribute_name(:responsable_contact), with: "0045-1234"
```

Comment:
A question: why is this change here? is there a reason?

Reply:
Yes, Rubocop was complaining because too long :)

Reply:
I would use the field's id (i.e. `group_responsible_contact` instead of `Group.human_attribute_name(:responsable_contact)` and then use input that is close to reality.

Which brings the question: should you have a validation on phone number? for example, that it has to be between 9-10 characters?